go/mysql/server.go

```diff
@@ -37,8 +37,14 @@ const (
 	DefaultServerVersion = "5.5.10-Vitess"
 
 	// timing metric keys
-	connectTimingKey = "Connect"
-	queryTimingKey   = "Query"
+	connectTimingKey  = "Connect"
+	queryTimingKey    = "Query"
+	versionSSL30      = "SSL 3.0"
```

Comment:
Since these are tags, it's risky to use spaces (or dots). Some value parsers may treat them as separators. Underscores are safe. Even `-` could cause trouble. Personally, I'd just go with SSL30, TLS10, etc.